nupic/research/frameworks/continuous_learning/dc_k_winners.py

```diff
@@ -0,0 +1,264 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
Is this file being used anywhere?

Reply:
it's the file where i'm attempting to update the duty cycles after each epoch/class. i haven't integrated it anywhere but it's something i'd like to do